mne/decoding/mixin.py

```diff
@@ -65,3 +65,34 @@ def set_params(self, **params):
                                      (key, self.__class__.__name__))
                 setattr(self, key, value)
         return self
+
+
+class VectorizerMixin(object):
+    """Mixin class for transformation of different data to fit scikit-learn
+       pipeline.
+    """
+
+    def transform(self, data, y=None):
```

Comment:
@agramfort I added a `transform` method similar following scikit-learn API. I didn't add `fit` because no values need to be calculated. Please let me know if I am proceeding correctly.